Watcher rebalance

MacHac opened the pull request:
I'm not calling this a nerf, because by some metrics watchers actually become more dangerous, but that's effectively what it is.

NEW SHIT:
Bolamine, the slowdown chemical.  Bolamine is made by applying one part frost oil to two parts space drugs.  The frost oil is consumed, and the space drugs become bolamine.  Bolamine functions as 'reverse meth'; It slowly eats stamina, and causes a slowdown effect.  Bolamine's effect can be mitigated using stimpacks and meth, and completely negated (at least temporarily) by Ephedrine and morphine, as well as any other chemical that gives the 'NOSLOWDOWN' flag.

This PR also adds the SLOWDOWN flag for other chemicals to potentially use.

OLD SHIT:
Basilisks and watchers have had their freeze beam replaced by an organic dart containing 5u bolamine.  This negates their ranged damage potential, but makes their slowdown slightly more potent and far longer lasting (as well as adding a stamina damage element).  In order to compensate for their loss of ranged damage, basilisk and watcher melee has been buffed from 9 shots to kill to 7 (on an unarmored human).

WHAT THIS MEANS:
-Cabbages will get less powerful in mining, because other races no longer have to worry about regenerating 40 burn damage after every watcher encounter.
-Watcher tendrils should no longer be nearly unassailable (although they remain dangerous).
-People should now hoard less burn patches
-Watcher slowdowns last far longer (on the order of 10 minutes for a large group of them), necessitating more careful play and survival capsule usage.  This also makes it much harder to raid tendrils, necessitating the use of stimpacks and such.
-While not lethal in and of itself, the stamina stun that large amounts of bolamine can cause is incredibly lethal if there's no availible shelter.  This means that watchers remain the deadliest motherfuckers in space.

WHY IT'S NECCESARY:
Let's run down a list of mobs with ranged damage.
-Colossus (rediculous megabeast, powerful drop)
-Ash Drake (rediculous megabeast, powerful drop)
-Bubblegum (rediculous megabeast, powerful drop)
-Syndicate Agents (they have machine guns, usually guarding something powerful)
-Alien Sentinels (less powerful than watcher, guarding alien queen and some loot)
-Watcher (drops two sinew and a single bone)

One of these things is not like the others.

Other than that, watchers tend to have absurd hitscan aiming capabilities at close range (which is where you're fighting them 90% of the time) and necessitate doing stupid things like bringing an ore box to eat shots, or tanking them out until you go crit from the burn damage.

Finally, taking freezing damage in lavaland is fucking stupid.

:cl:
tweak: While studying the biology of slain Watchers, a Nanotrasen scientist realized that it should be impossible to freeze to death in the middle of Lavaland.  Shortly thereafter, all Watchers in Lavaland switched to firing organic darts filled with a natural muscle relaxant.
rscadd: Bolamine can now be produced in chemistry by reacting Frost Oil with Space Drugs.  It slows down the user, and drains their stamina.
/:cl:

ShadowDeath6 commented:
don't make it frost oil, it's impossible to get outside xenobio

MacHac commented:
It might need more frost oil, now that I think about it.  After all, 30 units of frost oil (one bottle) makes 60 units of bolamine.  Dilute that with water to 180u, and you have 12 syringes full of 30s slowness.  That's easily enough time for any traitor worth a damn to close to melee range, or line up a shot. And there's 12 of them.

So you have to go to science and ask nicely.  Boo hoo.  It ain't easy being cheesy.

ShadowDeath6 commented:
>you have to go to science and ask nicely

I think you mean you can only ever get it in science because nobody in their right mind would EVER give you frost oil or a slow + stamina damage chem

Super3222 commented:
> don't make it frost oil, it's impossible to get outside xenobio

Ever heard of Botany?

Super3222 commented:
> don't make it frost oil, it's impossible to get outside xenobio

Talk to a botanist.

Stealthkibbler commented:
FYI Cabbages have it worse when fighting watchers because they take almost twice as much burn damage from the freezing.

Also the buff on its melee attacks is way too much, its going to make it impossible to kill it as an Ashwalker, even in a group.

Stamina damage isn't on its own, if you take melee damage then the crit threshold for stamina becomes exponentially smaller, the melee buff is going to destroy entire ash walker hunting parties on its own. I would recommend keeping it at 9.

MacHac commented:
If you have a cult buddy in science, or a fellow rev, bolamine is nearly as powerful as its namesake in terms of group antag conversion ability (and it can't be resisted out of, only metabolized or purged).  Is it really such a sin to reward cooperation between members of group antags?

ShadowDeath6 commented:
you can also slap them with a stunprod or bar neuro if that's still a thing which is 10^100 times more effective and less annoying for conversion considering stamina damage is the digital manifestation of the AIDS virus

MacHac commented:
The SOLE purpose for the existence of bolamine is as watcher poison.  I added the chemistry recipe as an afterthought, and it's intended to make the producer *gasp* cooperate with another department because otherwise it would be the most powerful conversion tool producible with a chem dispenser.  It's intended to supplement the use of other stuns, particularly melee stuns like the cult talisman or a baton.  Would you prefer I removed the recipe altogether?

ShadowDeath6 commented:
yes :^)

MacHac commented:
The recipe is fine, and unless someone else raises concerns with it, I see no reason to at least test it live and see if any problems come up.

As for @Stealthkibbler, I'm willing to compromise at 8 shots to kill, but their squishy HP pool means that virtually any quantity of salt lizards other than one can take the damn things down in moments.  As the PR stands right now, watchers are still less dangerous to walkers than a goliath in an enclosed space.

FluffySurvivor commented:
:+1:

oisin100 commented on code/modules/reagents/chemistry/holder.dm:
```diff
@@ -443,6 +443,14 @@ var/const/INJECT = 5 //injection
 		else
 			M.status_flags &= ~IGNORESLOWDOWN
 
+/datum/reagents/proc/check_slowdown(mob/M)  //SLOWDOWN is halfway in between GOTTAGOFAST and GOTTAGOREALLYFAST.  IGNORESLOWDOWN cancels it.
```
Should you actually call this somewhere?

MacHac commented:
fixed, oisin pls
